content/en/news/releases/1.1.x/announcing-1.1/change-notes/index.md

```diff
@@ -37,8 +37,8 @@ concise list of things you should know before upgrading your deployment to Istio
 
 - **Improved Multicluster Integration**. Consolidated the 1.0 `istio-remote`
   chart previously used for
-  [multicluster VPN](/docs/setup/install/multicluster/shared-vpn/) and
-  [multicluster split horizon](/docs/setup/install/multicluster/shared-gateways/) remote cluster installation
+  [multicluster VPN](/docs/setup/install/multicluster/shared/) and
```

Comment:
For old release notes and change notes we've recently adopted a convention of pointing broken links to the original location in the corresponding archive docs. I think that might be a good idea for these as well.
```
  [multicluster VPN](https://archive.istio.io/v1.1/docs/setup/kubernetes/install/multicluster/vpn/) and
  [multicluster split horizon](https://archive.istio.io/v1.1/docs/examples/multicluster/split-horizon-eds/) remote cluster installation
```